utils/node_local_test.py

```diff
@@ -3984,8 +3989,14 @@ def test_alloc_fail_copy(server, conf, wf):
 
     pool = server.get_test_pool()
     src_dir = tempfile.TemporaryDirectory(prefix='copy_src_',)
-    with open(join(src_dir.name, 'file'), 'w') as ofd:
-        ofd.write('hello')
+    sub_dir = join(src_dir.name, 'new_dir')
+    os.mkdir(sub_dir)
+    for f in range(5):
+        with open(join(sub_dir, 'file.{}'.format(f)), 'w') as ofd:
+            ofd.write('hello')
+
+    os.symlink('broken', join(sub_dir, 'broken'))
+    os.symlink('new_dir/file.0', 'link')
```

Comment:
This was a mistake on my part but you need this change as well, or it's making the link where the test is run, not in the source directory.
```suggestion
    os.symlink('new_dir/file.0', link(sub_dir, 'link'))
```